### PR TITLE
Update active_users data checks

### PR DESCRIPTION
**Pull request by lucia-vargas-a**
Implements [DENG-2355](https://mozilla-hub.atlassian.net/browse/DENG-2355) to update the data checks with the new naming for dau and daily_users metrics.
Solves warning checks failures. E.g. [For Desktop](https://workflow.telemetry.mozilla.org/dags/bqetl_analytics_aggregations/grid?search=bqetl_analytics_aggregations&dag_run_id=scheduled__2024-01-09T04%3A15%3A00%2B00%3A00&task_id=checks__warn_fenix_derived__active_users_aggregates__v3&tab=logs).

[DENG-2355]: https://mozilla-hub.atlassian.net/browse/DENG-2355?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/DENG-2357)

### CODE REVIEW

File: sql_generators/active_users/templates/fenix_checks.sql [hunk 1]
```diff
@@ -27,9 +27,9 @@ distinct_client_count_nightly_base AS (
   WHERE
     DATE(submission_timestamp) = @submission_date
     AND mozfun.norm.fenix_app_info("org_mozilla_fenix", client_info.app_build).channel = "nightly"
-    -- NOTE: the below two tables are marked as depricated inside the GLEAN dictionary
-    -- however, they are still considered when generating active_users_aggregates metrics
-    -- this is why they are being considered here.
+    -- NOTE: the below two tables are marked as deprecated inside the GLEAN dictionary.
```
Comment (gkabbz) [inline sql_generators/active_users/templates/fenix_checks.sql:30]: @lucia-vargas-a which two tables are deprecated? I was confused coz the only table I saw below was the `client_info.client_id`

**Review by gkabbz — APPROVED** [review]
Left a comment on one of the comments in the code.

File: sql_generators/active_users/templates/fenix_checks.sql (hunk 1 above)
Comment (lucia-vargas-a) [reply sql_generators/active_users/templates/fenix_checks.sql:30]: Tables org_mozilla_fenix_nightly_live.baseline_v1 &
org_mozilla_fennec_aurora_live.baseline_v1.  @kik-kik could you confirm, since you found this while building the checks?

**Review by kik-kik — APPROVED** [review]

File: sql_generators/active_users/templates/fenix_checks.sql (hunk 1 above)
Comment (kik-kik) [reply sql_generators/active_users/templates/fenix_checks.sql:30]: yes, so if you look at GLEAN dictionary you can see that some of the Fenix tables are marked as deprecated, however, it looks like they were still being used in our models / table (and still get a small amount of data). I'm pretty sure I've mentioned this before to you @gkabbz .

https://dictionary.telemetry.mozilla.org/apps/fenix?itemType=app_ids&page=1

I could swear there were apps marked as deprecated in the dictionary when I looked, but don't see it anymore. However, if you look at `org.mozilla.fennec.aurora` specifically the description says `Nightly channel of Firefox for Android users migrated to Fenix; delisted in June 2020.` So my understanding would be that this specific app was deprecated in June 2020. Of course, delisted does not mean everyone suddenly stopped using it so data could still be flowing in.

But found it:
https://web.archive.org/web/20230330013651/https://dictionary.telemetry.mozilla.org/apps/fenix?itemType=app_ids&page=1

So to be honest, I'm not sure now whether these app ids are indeed deprecated or not. I linked both of you a conversation on Slack where I try to get some insight.


**In short: I'm not sure if this comment is valid.**

Comment (kik-kik) [reply sql_generators/active_users/templates/fenix_checks.sql:30]: Based on Jan-Erik's response, those two tables are still deprecated.

Comment (kik-kik) [reply sql_generators/active_users/templates/fenix_checks.sql:30]: So, perhaps that's sort of true. The deprecation label has been removed in October via:
https://github.com/mozilla/probe-scraper/pull/640/files

Comment (lucia-vargas-a) [reply sql_generators/active_users/templates/fenix_checks.sql]: Thanks @kik-kik, I updated the comment to reflect the actual status. For both tables the data continues to be reported and collected.
